app/controllers/general.php

```diff
@@ -301,23 +301,21 @@
     /** @var Utopia\Logger\Logger $logger */
     /** @var Utopia\Logger\Log\Breadcrumb[] $loggerBreadcrumbs */
 
-    $user = null;
-    /** @var Appwrite\Database\Document $user */
-
-    try {
-        $user = $utopia->getResource('user');
-    } catch(\Throwable $th) {
-        // All good, user is optional information for logger
-    }
-
     $version = App::getEnv('_APP_VERSION', 'UNKNOWN');
     $route = $utopia->match($request);
 
     if($logger) {
         if($error->getCode() >= 500 || $error->getCode() === 0) {
+            try {
```

Comment:
@Meldiron please remind me why do we use try-catch here? If the user is not available should we get null or an empty user object?

Reply:
getResurce can throw error if JWT is invalid.

Reply:
In that case shouldn't we also throw 401 error? Can we implement the error handler on resource creation as part of utopia/fw?

Reply:
@christyjacob4 explained to me the issue here, as long as this is only and error handler thing, we ca definitely wait and find a more elegant solution later.